indico/modules/events/tracks/client/js/index.js

```diff
@@ -24,34 +23,51 @@
           }
         });
 
-        $trackList.sortable({
-          axis: 'y',
-          containment: 'parent',
-          cursor: 'move',
-          distance: 2,
-          handle: '.ui-i-box-sortable-handle',
-          items: '> li.track-row',
-          tolerance: 'pointer',
-          forcePlaceholderSize: true,
-          update: function() {
-            var sortedList = $trackList
-              .find('li.track-row')
-              .map(function() {
-                return $(this).data('id');
-              })
-              .get();
-
-            $.ajax({
-              url: $trackList.data('url'),
-              method: 'POST',
-              contentType: 'application/json',
-              data: JSON.stringify({sort_order: sortedList}),
-              complete: IndicoUI.Dialogs.Util.progress(),
-              error: handleAjaxError,
-            });
-          },
+        $(this).find('.track-list').sortable({
+            items: '.track-row',
```

Comment:
I think the indendation of this object is a bit off. This file uses 2-space indendation whereas this object's properties are indented with 4 spaces.